include/aikido/io/util.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef AIKIDO_IO_UTIL_HPP_
+#define AIKIDO_IO_UTIL_HPP_
+
+#include <aikido/io/CatkinResourceRetriever.hpp>
+#include <dart/dart.hpp>
+#include <dart/utils/urdf/DartLoader.hpp>
+
+using dart::dynamics::SkeletonPtr;
+
+namespace aikido {
+namespace io {
+
+const SkeletonPtr makeBodyFromURDF(
```

Comment:
Nit: Returning by const value is generally unnecessary. Let's just return `SkeletonPtr`.